fixes/add_histfactory_analyses.py

```diff
@@ -0,0 +1,64 @@
+import click
+import logging
+
+from celery import shared_task
+from flask import current_app
+from flask.cli import with_appcontext
+from invenio_db import db
+
+from hepdata.celery import dynamic_tasks
+from hepdata.config import HISTFACTORY_FILE_TYPE
+from hepdata.cli import fix
+from hepdata.ext.elasticsearch.api import reindex_batch
+from hepdata.modules.records.utils.common import is_histfactory
+from hepdata.modules.submission.api import get_latest_hepsubmission
+from hepdata.modules.submission.models import HEPSubmission
+
+logging.basicConfig()
+log = logging.getLogger(__name__)
+
+@fix.command()
+@with_appcontext
+@click.option('--batch-size', '-b', type=int, default=20,
+              help='Number of hepsubmission entries to check at a time.')
+@click.option('--synchronous', '-s', type=bool, default=False)
+def add_histfactory_analyses(batch_size, synchronous=False):
+    all_ids = db.session.query(HEPSubmission.id).order_by(HEPSubmission.id).all()
+
```

Comment:
Good point...though you'd then have to filter out data resources belonging to tables so not sure how much difference it would make in the end. Happy to leave it if you are.